src/Compilers/CSharp/Portable/CommandLine/CommandLineParser.cs

```diff
@@ -1039,7 +1039,7 @@ public new CSharpCommandLineArguments Parse(IEnumerable<string> args, string bas
                 preprocessorSymbols: defines.ToImmutableAndFree(),
                 documentationMode: parseDocumentationComments ? DocumentationMode.Diagnose : DocumentationMode.None,
                 kind: SourceCodeKind.Regular
-            );
+            ).WithFeatures(features.ToImmutableDictionary(feature => feature, feature => "true"));
```

Comment:
@khyperia given that this PR is already changing `CSharpParseOptions` I would just go ahead and do it now.